pkg/postage/postagecontract/contract.go

```diff
@@ -263,49 +263,49 @@ func (c *postageContract) getBalance(ctx context.Context) (*big.Int, error) {
 	return abi.ConvertType(results[0], new(big.Int)).(*big.Int), nil
 }
 
-func (c *postageContract) CreateBatch(ctx context.Context, initialBalance *big.Int, depth uint8, immutable bool, label string) ([]byte, error) {
+func (c *postageContract) CreateBatch(ctx context.Context, initialBalance *big.Int, depth uint8, immutable bool, label string) (string, []byte, error) {
```

Comment:
ideally, instead of a string, it should return `common.Hash` type

Reply:
So I initially set it to `common.Hash`  but thought we only going to use string. Any specific reason for `common.Hash`?

Reply:
it's more specific, easier to read/understand, similar to how we return swarm.Address instead of []byte